src/app/components/stacks-transaction-item/stacks-transaction-item.tsx

```diff
@@ -78,19 +99,40 @@ export function StacksTransactionItem({
   const txIcon = transaction ? <StacksTransactionIcon transaction={transaction} /> : icon;
   const txTitle = transaction ? getTxTitle(transaction) : title || '';
   const txValue = transaction ? getTxValue(transaction, isOriginator) : value;
-  const increaseFeeButton = (
-    <IncreaseFeeButton
+
+  const actionButtonGroup = [
+    { label: 'Cancel', action: 'cancel' },
+    { label: 'Increase Fee', action: 'increaseFee' },
+  ].map(({ label, action }) => (
+    <TransactionActionButton
+      key={label}
+      icon={
+        action === 'cancel' ? (
+          <CloseIcon color="stacks" variant="small" height={12} />
```

Comment:
Do you need `height` on this icon?

Reply:
> Do you need `height` on this icon?

So if you check out the initial comments in the previous PR @314159265359879 mentioned about the cancel icon size being bigger than the increase fee icon. Even though i set both to small variant the cancel one was being bigger . I think the svg is bigger for `close` icon. Hence i had to override it